src/ioctls/mod.rs

```diff
@@ -223,6 +223,7 @@ impl CpuId {
 /// threads as raw pointers do not implement `Send` and `Sync`.
 pub struct KvmRunWrapper {
     kvm_run_ptr: *mut u8,
+    size: usize,
```

Comment:
nit: I'd call it something other than `size` (`mapping_size`?), because it's not obvious to someone who doesn't know/care about the underlying `mmap` that we need to know the mapping size for later.

Reply:
I agree, otherwise LGTM 👍

Reply:
Done.